homeassistant/components/media_player/__init__.py

```diff
@@ -72,6 +72,7 @@
 SUPPORT_TURN_ON = 128
 SUPPORT_TURN_OFF = 256
 SUPPORT_PLAY_MEDIA = 512
+SUPPORT_VOLUME_BUTTONS = 1024
```

Comment:
I don't think volume buttons describes the functionality appropriately. The fact that they are represented as buttons is a UI concern, not a support flag concern. What about `SUPPORT_VOLUME_STEP` to indicate it supports modifying the volume in steps.